test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -1544,6 +1544,46 @@ public void CopyColumnsOnnxTest()
             Done();
         }
 
+        [Fact]
+        public void SelectiveExportOnnxTest()
+        {
+            var mlContext = new MLContext(seed: 1);
+
+            var trainDataPath = GetDataPath(TestDatasets.generatedRegressionDataset.trainFilename);
+            var dataView = mlContext.Data.LoadFromTextFile<AdultData>(trainDataPath,
+                separatorChar: ';',
+                hasHeader: true);
+
+            var mlpipeline = mlContext.Transforms.CopyColumns("Target1", "Target");
+            var onnxFileName = "copycolumns.onnx";
+
+            var mlmodel = mlpipeline.Fit(dataView);
+
+            var onnxModelPath = GetOutputPath(onnxFileName);
+            using (var stream = File.Create(onnxModelPath))
+            {
+                mlContext.Model.ConvertToOnnx(mlmodel, dataView, stream, "Target1");
```

Comment:
I'm just curious: would there be any [performance] difference from using this new API providing the "Target1" "column to keep" versus NOT using this new API but adding a DropColumns transformer, that is configured to only keep the "Target1" column, and _then_ exporting it to Onnx? From looking into the code of the PR I'd feel it'd have the same outcome?

Reply:
For the record, it seems the main difference is from a usability point of view, as it might not be very intuitive to add a DropColumns for this purpose, as well that the user might want to _not_ add the drop columns to their pipeline (so as to not modify their output IDataView), while still being able to export to onnx only a subset of the outputs.